extensions/oidc-token-propagation-reactive/deployment/src/main/java/io/quarkus/oidc/token/propagation/reactive/OidcTokenPropagationReactiveBuildTimeConfig.java

```diff
@@ -13,4 +13,18 @@ public class OidcTokenPropagationReactiveBuildTimeConfig {
      */
     @ConfigItem(defaultValue = "true")
     public boolean enabled;
+
+    /**
+     * Whether the token propagation is enabled during the `SecurityIdentity` augmentation.
+     *
+     * For example, you may need to use a REST client from `SecurityIdentityAugmentor`
+     * to propagate the current token to acquire additional roles for the `SecurityIdentity`.
+     *
+     * Note, this feature rely on a duplicated context. More information about Vert.x duplicated
```

Comment:
```suggestion
     * Note, this feature relies on a duplicated context. More information about Vert.x duplicated
```

Reply:
@michalvavrik Interesting idea about linking to the guides from JavaDocs - do we do somewhere else in Quarkus thought ? Say the Doc team changes `duplicated-context` to `duplicated-context-concept` and the link gets broken. May be we can simplify with `Note, this feature relies on a duplicated Vert.x context.` and users, if interested, can type that in Google and find the guide ?

Reply:
well @sberyozkin , they can, but I think it is better this way because you have there link and on one click user gets very nice guide (I say it because I like the guide). 

> do we do it somewhere else in Quarkus thought ?

I can see it on Hibernate guides, let's see `io.quarkus.hibernate.search.orm.elasticsearch.runtime.HibernateSearchElasticsearchBuildTimeConfigPersistenceUnit#backgroundFailureHandler` and then you can check https://quarkus.io/guides/all-config#quarkus-hibernate-search-orm-elasticsearch_quarkus.hibernate-search-orm.background-failure-handler for how it looks. It will work for this PR as well when linked guide gets released.

>  Say the Doc team changes duplicated-context to duplicated-context-concept and the link gets broken. 

But how does it differ to situation when you refer to guide from different guide. I don't know how automated tools do it, but personally I simply fulltext search for what I changed (`xref:duplicated-context`) and if I find it, I have to change it.

IMO current state is better. But it is not a deal breaker, WDYT? Convinced?

Reply:
@michalvavrik Sure, lets do it and see how it goes.

Reply:
@michalvavrik Please commit 2 minor suggestions and we can merge it